content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -57,10 +57,25 @@ This will be set only if you are using a `cloudprovider`. This setting is handy
 if you want to target certain workloads to certain instance types, but typically you want
 to rely on the Kubernetes scheduler to perform resource-based scheduling. You should aim to schedule based on properties rather than on instance types (for example: require a GPU, instead of requiring a `g2.2xlarge`).
 
+{{< note >}} Starting in v1.17, this label is deprecated in favor of [node.kubernetes.io/instance-type](#nodekubernetesioinstance-type). {{< /note >}}
+
+## node.kubernetes.io/instance-type {#nodekubernetesioinstance-type}
+
+Example: `node.kubernetes.io/instance-type=m3.medium`
+
+Used on: Node
+
+The Kubelet populates this with the instance type as defined by the `cloudprovider`.
+This will be set only if you are using a `cloudprovider`. This setting is handy
```

Comment:
Hello @andrewsykim ,
line 68: As I recall, this page uses `this` to mean labels/annotations/taints? Does this section apply to all terms?
line 69: This will be set ... `cloudprovider` --meaning that the instance-type will be set if the user has defined the `providerID`?

Reply:
@kbhawkey this is an exact copy of what was already written for the old label (`beta.kubernetes.io/instance-type`) which I copied for consistency reasons. I can update both based on our feedback but would prefer to do this in a separate PR. Let me know what you think is best.

Reply:
Hello @andrewsykim , OK. Sure, I found the statements about `cloudprovider` confusing.

The notes are good, though to look like the other deprecated types, could you add _deprecated_ to the headers (like the other types that are deprecated), instead of a note referencing v1.17?
For example:
```
## beta.kubernetes.io/instance-type (deprecated)
## failure-domain.beta.kubernetes.io/region (deprecated)
## failure-domain.beta.kubernetes.io/zone (deprecated)
```
Thanks!

Reply:
good catch! updated :)

Reply:
I think so because it's not always clear what version of the docs you're reading, stating it explicitly is helpful so users know when the replacement labels were added.

Reply:
OK